docs/book/how-to/setting-up-a-project-repository/set-up-repository.md

```diff
@@ -34,7 +34,21 @@ While it doesn't matter how you structure your ZenML project, here is a recommen
 └── run.py
 ```
 
-All ZenML [Project templates](using-project-templates.md#generating-project-from-a-project-template) are modeled around this basic structure.
+All ZenML [Project
+templates](using-project-templates.md#generating-project-from-a-project-template)
+are modeled around this basic structure. The `steps` and `pipelines` folders
+contain the steps and pipelines defined in your project. If your project is
+simpler you can also just keep your steps at the top level of the `steps` folder
+without the need so structure them in subfolders.
+
+{% hint style="info" %}
+It might also make sense to register your repository as a code repository. These
+enable ZenML to keep track of the code version that you use for your pipeline
+runs. Additionally, running a pipeline that is tracked in a registered code
+repository can speed up the Docker image building for containerized stack
+components by eliminating the need to rebuild Docker images each time you change
+one of your source code files. Learn more about these in <a href="https://docs.zenml.io/how-to/setting-up-a-project-repository/connect-your-git-repository">connecting your Git repository</a>.
```

Comment:
this HREF is a docs.zenml.io link, but should rather be a markdown link in a relative path?

Reply:
If I'm not mistaken, gitbook doesn't render the link if you put a markdown link inside these info and warning boxes. I always do them as HTML links, at which point you can't refer to markdown any more etc. I'll see if gitbook has fixed this.

Reply:
Oh looks like they fixed it. I'll update.